pom.xml

```diff
@@ -58,6 +59,37 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-log4j2</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>org.apache.logging.log4j</groupId>
+					<artifactId>log4j-slf4j-impl</artifactId>
+				</exclusion>

```

Comment:
I don't think it's necessary to exclude these transitive dependencies. Adding them as explicit dependencies with explicit versions should be sufficient to override the transitive versions.

Reply:
Ok, I was wondering if this was necessary or not.  Had a read through http://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html and you're right, it's enough that we're stating the versions later on.